stories/1-Dropdown.stories.tsx

```diff
@@ -55,40 +98,146 @@ storiesOf("Dropdown", module).add("Dropdown States", () => {
               hasError={true}
             />
           </FormField>
+        </div>
+      )}
+    </StateProvider>
+  ))
+  .add("Dropdown Menu", () => (
+    <StateProvider initialState={initialState}>
+      {(state, setState) => (
+        <StoryFragment>
+          <div style={{maxWidth: "350px"}}>
+            <FormField>
+              <Dropdown
+                role={"menu"}
+                options={state.options}
+                placeholder={"Select Language"}
+                onSelect={(option) => setState({...state, selectedOption: option})}
+                selectedOption={state.selectedOption}
+                hasDeselectOption={false}
+                header={
+                  <div>
+                    <span className={"circle"} />
+                    <span className={"dropdown-header"}>{"Language Menu"}</span>
+                    <style>
+                      {`
+                        .circle {
+                          height: 8px;
+                          width: 8px;
+                          background-color: blue;
+                          border-radius: 50%;
+                          display: inline-block;
+                        }
+                        
+                        .dropdown-header {
+                          color: black;
+                          margin-left: 16px;
+                        }
+                        `}
+                    </style>
+                  </div>
+                }
+              />
+            </FormField>
+          </div>
 
-          <FormField label={"Your Language - no options"}>
+          <span>{`Selected Language: ${
+            state.selectedOption ? state.selectedOption.title : "-"
+          }`}</span>
+        </StoryFragment>
+      )}
+    </StateProvider>
+  ))
+  .add("Dropdown Options With Subtitles", () => (
+    <StateProvider initialState={initialStateWithSubtitle}>
+      {(state, setState) => (
+        <div style={{maxWidth: "350px"}}>
+          <FormField label={"Web Courses"}>
             <Dropdown
               role={"listbox"}
-              options={[]}
+              options={state.options}
               placeholder={"Select Language"}
               onSelect={(option) => setState({...state, selectedOption: option})}
               selectedOption={state.selectedOption}
             />
           </FormField>
-
-          <FormField label={"Your Language"}>
+        </div>
+      )}
+    </StateProvider>
+  ))
+  .add("Dropdown With Customized Options", () => (
+    <StateProvider initialState={initialStateWithContext}>
```

Comment:
This doesn't show how one would use "Customized Options" but shows the use of `context`